tools/run_tests/run_performance_tests.py

```diff
@@ -403,6 +406,10 @@ def finish_qps_workers(jobs):
                   action='store_const',
                   const=True,
                   help='Run netperf benchmark as one of the scenarios.')
+argp.add_argument('--server_cpu_load',
```

Comment:
Could we just make this an integer argument? Maybe we get interested in 17% CPU load